tests/integration/test_task_worker.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from cabbage.tasks import TaskManager
+from cabbage.task_worker import Worker
+
+
+@pytest.fixture()
+def results():
+    return {"sums": [], "products": []}
+
+
+@pytest.fixture()
+def task_manager(connection, results):
+    manager = TaskManager(connection)
+
+    @manager.task(queue="sum_queue")
+    def sum_task(_, a, b):
+        results["sums"].append(a + b)
+
+    @manager.task(queue="sum_queue")
+    def stop_task(_):
+        raise KeyboardInterrupt()
```

Comment:
I may be off but I thought unhandled sigint produced KeyboardInterrup, and we're handling sigint, so in real life, no KeyboardInterrupt will be raised, and this code will only be used if a task manually raises keyboard interrupt, no ?